gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -47,7 +47,8 @@ public class GcpLogSink implements Sink {
 
   // TODO(DNVindhya): Make cloud logging service a configurable value
   private static final String SERVICE_TO_EXCLUDE = "google.logging.v2.LoggingServiceV2";
-  private static final String DEFAULT_LOG_NAME = "grpc";
+  private static final String DEFAULT_LOG_NAME =
+      "microservices.googleapis.com%2Fobservability%2Fgrpc";
```

Comment:
Couple of things about this change:

- I am surprised this change does not need any unit test change (@DNVindhya we should have a unit test to verify this log name?)
- we don't have any interop tests to verify with CloudLogging. Anyway we can verify and manually and document here?

Reply:
- Yes, we should verify that our log entries always use this log name, checks added in our unit test.
- Run a manual test with Vindhya's help. Here's the log name shown in GCP cloud logging GUI.
- 
![image](https://user-images.githubusercontent.com/20803483/167034622-62df3d30-3c3d-4312-b760-784425b40de7.png)

Reply:
Interesting. If the log-name appears without percent decoding (i.e. as `microservices.googleapis.com%2Fobservability%2Fgrpc` instead of `microservices.googleapis.com/observability/grpc`) then what's the use for the end user? I thought the `%2F` is only for the intermediate plumbing thru the API but in the dashboard I should expect to see a properly formatted string like `projects/grpc-testing/logs/microservices.googleapis.com/observability/grpc` . Am I missing something?